pkg/reconciler/accessor/istio/virtualservice.go

```diff
@@ -64,7 +65,9 @@ func ReconcileVirtualService(ctx context.Context, owner kmeta.Accessor, desired
 		return nil, err
 	} else if !metav1.IsControlledBy(vs, owner) {
 		// Surface an error in the ClusterIngress's status, and return an error.
-		return nil, fmt.Errorf("ingress: %q does not own VirtualService: %q", owner.GetName(), name)
+		return nil, kaccessor.NewAccessorError(
+			fmt.Errorf("ingress: %q does not own VirtualService: %q", owner.GetName(), name),
```

Comment:
Should the accessor assume this is used in the context of ingress?

Reply:
No. fixed it.